packages/react-components/react-alert/src/components/Alert/useAlert.tsx

```diff
@@ -50,6 +53,8 @@ export const useAlert_unstable = (props: AlertProps, ref: React.Ref<HTMLElement>
     },
     root: getNativeElementProps('div', {
       ref,
+      role: defaultRole,
+      'aria-live': 'assertive',
```

Comment:
@rohitpagariya looks like the figma still needs to be updated. I'd prefer not to hold up the PR based on that, though. Are you OK with the change going in while design works on the updating the spec?

Reply:
Yes, please go ahead and merge the PR :)

Reply:
what is the behavior for assertive with role='status' - is that being considered by the screen readers? As far as I remember, alert is by default assertive. Do we need to make status assertive as well?

Reply:
my worry is that if this ends up being used for confirmations (something like 'logs uploaded', making it assertive might distract the user from what they are actually doing. I would much rather make it as an opt-in on the component level and let the product decide.

As far as I can see from the docs site, this is currently not the component that would be managing the toast nature, this appears to be just the content of the toast - @rohitpagariya can you please confirm if this is the plan?

Reply:
Even for something like "logs updated", I think `aria-live="assertive"` would be the better choice. Unless there are frequent interruptions, the feedback I've heard is that success/confirmation messages as assertive is a good thing.

The use case where I would use `aria-live="polite"` is more when something fully doesn't matter whether the user notices it or not. For a default, based on the design of Toast, I think polite isn't a good fit. As @khmakoto mentioned, if an author has a specific use case where the toasts are frequent or particularly unimportant, they can customize the `aria-live` value (though in that case the visual distraction might be its own separate issue).

Reply:
With the toasts feature, Teams will have two temporal notification systems - banners & toasts. It's not clear to me when an app should use banners vs toasts. I will follow up with design / PM team on clear & written guidance.

Today, banners in Teams use role="alert" & "aria-live=polite" by default. And these aria attributes don't change based on the intent. 

To Juraj's question, the current plan is to use Alert to render the content of the toast. I have already built a ToastProvider component to manage the incoming & outgoing toasts and ToastRenderer component to handle animations, auto dismiss & rendering the toast data via the Alert component. 

The v0 Alert component also sets role="alert" & "aria-live=polite" by default

Reply:
Toasts are meant to be used for transient notifications, typically in response to an action the user just took. And so the most typical use of the toast will be something like "download completed" or "logs uploaded," thus polite is appropriate because one of the top complaints from screen readers is too much information and constant distractions. Let's cut down on those distractions where we can -- in fact, Brett said that many screen reader users even pay for a Teams add-on which turns off most of the Teams notifications from being narrated (!).

If there is something that the user needs to know, such as their download couldn't be completed, that can be an error and given priority so that users can take action.

The decision tree for banner vs toast usage can be found here: https://www.figma.com/file/HQ4nlkneIZvWUFzehrQoEL/Toast?node-id=2075%3A221556